README.md

```diff
@@ -83,44 +85,35 @@ This script will:
 2. Ensure you have the desired version of the plugin you want to install, get it from github releases 
 
    ```
-   curl -s https://api.github.com/repos/netdata/netdata-grafana-datasource-plugin/releases/latest \
-   jq -r '.assets[] | select(.name|match("zip$")) | .browser_download_url'
+   wget `curl -s https://api.github.com/repos/netdata/netdata-grafana-datasource-plugin/releases/latest | jq -r '.assets[] | select(.name|match("zip$")) | .browser_download_url'`
    ```
 
-3. Grafana plugins, by default, should be under /var/lib/grafana/plugins. Create a folder for netdata inside the container. Enter in an interactive session
+3. Copy the contents of the Netdata data source plugin to Grafana plugins directory, by default /var/lib/grafana/plugins
 
    ```
-   docker exec -ti grafana bash
-   mkdir /var/lib/grafana/plugins/netdata
+   unzip netdata-datasource-<version_number>.zip
+   docker cp netdata-datasource grafana:/var/lib/grafana/plugins/
    ```
 
-4. Copy the contents of the Netdata data source plugin to the new plugin folder created on step 2
-
+4. Restart grafana container
+   
    ```
-   unzip netdata-datasource-<version_number>.zip
-   docker cp netdata-datasource grafana:/var/lib/grafana/plugins/netdata/
+   docker restart grafana
    ```
 
 ### Linux (local)
 
 1. Ensure you have the desired version of the plugin you want to install, get it from github releases 
 
    ```
-   curl -s https://api.github.com/repos/netdata/netdata-grafana-datasource-plugin/releases/latest \
-   jq -r '.assets[] | select(.name|match("zip$")) | .browser_download_url'
+   wget `curl -s https://api.github.com/repos/netdata/netdata-grafana-datasource-plugin/releases/latest | jq -r '.assets[] | select(.name|match("zip$")) | .browser_download_url'`
    ```
 
-2. Grafana plugins, by default, should be under /var/lib/grafana/plugins. Create a folder for netdata
-
-   ```
-   mkdir /var/lib/grafana/plugins/netdata
-   ```
-
-3. Copy the contents of the Netdata data source plugin to the new plugin folder created on step 2
+2. Copy the contents of the Netdata data source plugin to Grafana plugins directory, by default /var/lib/grafana/plugins
 
    ```
    unzip netdata-datasource-<version_number>.zip
-   cp -rf netdata-datasource/ /var/lib/grafana/plugins/netdata
+   cp -rf netdata-datasource/ /var/lib/grafana/plugins
```

Comment:
should be `netdata-datasource` with slash, because it copies only the content of dir